### PR TITLE
release-2024-08-26

### PR DESCRIPTION
**Pull request by batiati**
~Pending on https://github.com/tigerbeetle/tigerbeetle/pull/2240.~

### CODE REVIEW

File: src/scripts/release.zig [hunk 1]
```diff
@@ -61,8 +61,8 @@ pub fn main(shell: *Shell, gpa: std.mem.Allocator, cli_args: CliArgs) !void {
     // If you change this, make sure to change the validation code in release_validate.zig!
     const release_triple = multiversioning.ReleaseTriple{
         .major = 0,
-        .minor = 15,
-        .patch = @intCast(cli_args.run_number - 188),
+        .minor = 16,
+        .patch = 0, // TODO: Read the version number from `changelog.md`.
```
Comment (sentientwaffle) [inline src/scripts/release.zig:65]: `CliArgs.run_number` can be removed (or just add a TODO to remove it, since this is a release PR).

Comment (matklad) [reply src/scripts/release.zig:65]: I'd leave it as a todo! I am going to refactor this whole thing anyway by the next monday

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,61 @@
...
+  `Account`s and `Transfer`s from historical events.
+
+- [#2220](https://github.com/tigerbeetle/tigerbeetle/pull/2220),
+  [#2237](https://github.com/tigerbeetle/tigerbeetle/pull/2237),
+  [#2238](https://github.com/tigerbeetle/tigerbeetle/pull/2238),
+  [#2239](https://github.com/tigerbeetle/tigerbeetle/pull/2239)
+
+  Allow `Transfer`s with `amount=0` and change behavior for _balancing_ and _pending_ transfers,
+  introducing the constant `AMOUNT_MAX` to replace the use of the zero sentinel when representing
+  the maximum/original value in such cases.
```
Comment (sentientwaffle) [inline CHANGELOG.md]:
```suggestion
  Allow `Transfer`s with `amount=0` and change behavior for _balancing_ and _post-pending_ transfers,
  introducing the constant `AMOUNT_MAX` to replace the use of the zero sentinel when representing
  the maximum/original value in such cases. Note that this is a [**breaking change**](https://github.com/tigerbeetle/tigerbeetle/issues/2231#issuecomment-2305132591)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,61 @@
 # TigerBeetle Changelog
 
+## 2024-08-26
+
```
Comment (sentientwaffle) [inline CHANGELOG.md:7]: Maybe add a note referencing https://github.com/tigerbeetle/tigerbeetle/issues/2231, that people can subscribe to that issue to be notified about breaking client stuff?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,65 @@
...
+  Add the new `imported` flag to allow user-defined timestamps when creating
+  `Account`s and `Transfer`s from historical events.
+
+- [#2220](https://github.com/tigerbeetle/tigerbeetle/pull/2220),
+  [#2237](https://github.com/tigerbeetle/tigerbeetle/pull/2237),
+  [#2238](https://github.com/tigerbeetle/tigerbeetle/pull/2238),
+  [#2239](https://github.com/tigerbeetle/tigerbeetle/pull/2239)
+
+  Allow `Transfer`s with `amount=0` and change behavior for _balancing_ and _pending_ transfers,
```
Comment (sentientwaffle) [inline CHANGELOG.md]:
```suggestion
  Allow `Transfer`s with `amount=0` and change behavior for _balancing_ and _post-pending_ transfers,
```

**Review by sentientwaffle — DISMISSED** [review]

**Review by sentientwaffle — APPROVED** [review]

**Pull request merged**